### PR TITLE
Add broadcast test

### PR DESCRIPTION
**Pull request by wcy666103**
add test case to avoid https://github.com/apache/dubbo/issues/14280

### CODE REVIEW

File: 2-advanced/dubbo-smples-broadcast/src/main/java/org/apache/dubbo/samples/broadcast/EmbeddedZooKeeper.java [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+ * Helper class to start an embedded instance of standalone (non clustered) ZooKeeper.
+ * <p>
+ * NOTE: at least an external standalone server (if not an ensemble) are recommended, even for
+ * {@link org.springframework.xd.dirt.server.singlenode.SingleNodeApplication}
+ *
+ * @author Patrick Peralta
+ * @author Mark Fisher
+ * @author David Turanski
+ */
+public class EmbeddedZooKeeper implements SmartLifecycle {
```
Comment (CrazyHZM) [inline 2-advanced/dubbo-smples-broadcast/src/main/java/org/apache/dubbo/samples/broadcast/EmbeddedZooKeeper.java:45]: It is not recommended to maintain EmbeddedZooKeeper.

File: 2-advanced/dubbo-smples-broadcast/src/main/java/org/apache/dubbo/samples/broadcast/BroadcastProvider.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+package org.apache.dubbo.samples.broadcast;
+
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+
+import java.util.concurrent.CountDownLatch;
+
+public class BroadcastProvider {
+
+    public static void main(String[] args) throws Exception {
+//        new EmbeddedZooKeeper().start();
```
Comment (CrazyHZM) [inline 2-advanced/dubbo-smples-broadcast/src/main/java/org/apache/dubbo/samples/broadcast/BroadcastProvider.java:29]: It's no use.

**Review by CrazyHZM — CHANGES_REQUESTED** [review]

File: 2-advanced/dubbo-smples-broadcast/src/main/java/org/apache/dubbo/samples/broadcast/EmbeddedZooKeeper.java (hunk 1 above)
Comment (wcy666103) [reply 2-advanced/dubbo-smples-broadcast/src/main/java/org/apache/dubbo/samples/broadcast/EmbeddedZooKeeper.java]: already delete

File: 2-advanced/dubbo-smples-broadcast/src/main/java/org/apache/dubbo/samples/broadcast/BroadcastProvider.java (hunk 2 above)
Comment (wcy666103) [reply 2-advanced/dubbo-smples-broadcast/src/main/java/org/apache/dubbo/samples/broadcast/BroadcastProvider.java]: done

**Comment by wcy666103** [comment]
The failing ci has nothing to do with my change

image

**Pull request merged**